langchain4j-github-models/pom.xml

```diff
@@ -24,120 +24,7 @@
         <dependency>
             <groupId>com.azure</groupId>
             <artifactId>azure-ai-inference</artifactId>
-            <version>1.0.0-beta.1</version>
-            <exclusions>
```

Comment:
These exclusions were there to ensure dependency convergency, I see that this problem still exists:

```
Rule 0: org.apache.maven.enforcer.rules.dependency.DependencyConvergence failed with message:
Failed while enforcing releasability.

Dependency convergence error for io.netty:netty-resolver-dns:jar:4.1.109.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns-native-macos:jar:osx-x86_64:4.1.109.Final:compile
          +-io.netty:netty-resolver-dns-classes-macos:jar:4.1.109.Final:compile
            +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.projectreactor.netty:reactor-netty-core:jar:1.0.45:compile
          +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-resolver-dns:jar:4.1.111.Final:compile


Dependency convergence error for io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.projectreactor.netty:reactor-netty-core:jar:1.0.45:compile
          +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.109.Final:compile


Dependency convergence error for org.opentest4j:opentest4j:jar:1.2.0 paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-org.junit.jupiter:junit-jupiter-api:jar:5.9.3:compile
        +-org.opentest4j:opentest4j:jar:1.2.0:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-org.junit.jupiter:junit-jupiter-engine:jar:5.10.0:compile
        +-org.junit.platform:junit-platform-engine:jar:1.10.0:compile
          +-org.opentest4j:opentest4j:jar:1.3.0:compile


Dependency convergence error for io.netty:netty-codec-http2:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http2:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-codec-http2:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-codec-http2:jar:4.1.111.Final:compile


Dependency convergence error for io.netty:netty-resolver:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-resolver:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
          +-io.netty:netty-resolver:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-resolver:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-resolver:jar:4.1.111.Final:compile


Dependency convergence error for io.netty:netty-codec:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-codec:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-codec:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-codec-socks:jar:4.1.110.Final:compile
          +-io.netty:netty-codec:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http:jar:4.1.110.Final:compile
        +-io.netty:netty-codec:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http2:jar:4.1.110.Final:compile
        +-io.netty:netty-codec:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-codec:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-codec-dns:jar:4.1.109.Final:compile
            +-io.netty:netty-codec:jar:4.1.109.Final:compile


Dependency convergence error for io.netty:netty-buffer:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
          +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-codec-socks:jar:4.1.110.Final:compile
          +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec:jar:4.1.110.Final:compile
        +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http:jar:4.1.110.Final:compile
        +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http2:jar:4.1.110.Final:compile
        +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
        +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
        +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-classes-epoll:jar:4.1.110.Final:compile
          +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.110.Final:compile
        +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-classes-kqueue:jar:4.1.110.Final:compile
          +-io.netty:netty-buffer:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-buffer:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-codec-dns:jar:4.1.109.Final:compile
            +-io.netty:netty-buffer:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-buffer:jar:4.1.111.Final:compile


Dependency convergence error for io.netty:netty-transport-native-unix-common:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-classes-epoll:jar:4.1.110.Final:compile
          +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-classes-kqueue:jar:4.1.110.Final:compile
          +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns-native-macos:jar:osx-x86_64:4.1.109.Final:compile
          +-io.netty:netty-resolver-dns-classes-macos:jar:4.1.109.Final:compile
            +-io.netty:netty-transport-native-unix-common:jar:4.1.109.Final:compile


Dependency convergence error for io.netty:netty-transport:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-codec-socks:jar:4.1.110.Final:compile
          +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http2:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-classes-epoll:jar:4.1.110.Final:compile
          +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-classes-kqueue:jar:4.1.110.Final:compile
          +-io.netty:netty-transport:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-transport:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-codec-dns:jar:4.1.109.Final:compile
            +-io.netty:netty-transport:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-transport:jar:4.1.111.Final:compile


Dependency convergence error for io.netty:netty-common:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-resolver:jar:4.1.110.Final:compile
          +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
        +-io.netty:netty-transport:jar:4.1.110.Final:compile
          +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-codec-socks:jar:4.1.110.Final:compile
          +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-buffer:jar:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec:jar:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http:jar:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http2:jar:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-unix-common:jar:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-classes-epoll:jar:4.1.110.Final:compile
          +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.110.Final:compile
        +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.110.Final:compile
        +-io.netty:netty-transport-classes-kqueue:jar:4.1.110.Final:compile
          +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-common:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-codec-dns:jar:4.1.109.Final:compile
            +-io.netty:netty-common:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns-native-macos:jar:osx-x86_64:4.1.109.Final:compile
          +-io.netty:netty-resolver-dns-classes-macos:jar:4.1.109.Final:compile
            +-io.netty:netty-common:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-common:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-common:jar:4.1.111.Final:compile


Dependency convergence error for io.netty:netty-handler-proxy:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.projectreactor.netty:reactor-netty-core:jar:1.0.45:compile
          +-io.netty:netty-handler-proxy:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-handler-proxy:jar:4.1.111.Final:compile


Dependency convergence error for org.junit.jupiter:junit-jupiter-api:jar:5.9.3 paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-org.junit.jupiter:junit-jupiter-api:jar:5.9.3:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-org.junit.jupiter:junit-jupiter-engine:jar:5.10.0:compile
        +-org.junit.jupiter:junit-jupiter-api:jar:5.10.0:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-org.junit.jupiter:junit-jupiter-params:jar:5.10.0:compile
        +-org.junit.jupiter:junit-jupiter-api:jar:5.10.0:compile


Dependency convergence error for io.netty:netty-codec-http:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler-proxy:jar:4.1.110.Final:compile
        +-io.netty:netty-codec-http:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http2:jar:4.1.110.Final:compile
        +-io.netty:netty-codec-http:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-codec-http:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-codec-http:jar:4.1.111.Final:compile


Dependency convergence error for io.netty:netty-handler:jar:4.1.110.Final paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-handler:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http:jar:4.1.110.Final:compile
        +-io.netty:netty-handler:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.netty:netty-codec-http2:jar:4.1.110.Final:compile
        +-io.netty:netty-handler:jar:4.1.110.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.netty:netty-resolver-dns:jar:4.1.109.Final:compile
          +-io.netty:netty-handler:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-http-netty:jar:1.15.5:compile
      +-io.projectreactor.netty:reactor-netty-http:jar:1.0.45:compile
        +-io.projectreactor.netty:reactor-netty-core:jar:1.0.45:compile
          +-io.netty:netty-handler:jar:4.1.109.Final:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-com.azure:azure-core-http-vertx:jar:1.0.0-beta.22:compile
        +-io.vertx:vertx-core:jar:4.5.10:compile
          +-io.netty:netty-handler:jar:4.1.111.Final:compile


Dependency convergence error for org.junit.platform:junit-platform-commons:jar:1.9.3 paths to dependency are:
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-org.junit.jupiter:junit-jupiter-api:jar:5.9.3:compile
        +-org.junit.platform:junit-platform-commons:jar:1.9.3:compile
and
+-dev.langchain4j:langchain4j-github-models:jar:0.36.0-SNAPSHOT
  +-com.azure:azure-ai-inference:jar:1.0.0-beta.2:compile
    +-com.azure:azure-core-test:jar:1.27.0-beta.2:compile
      +-org.junit.jupiter:junit-jupiter-engine:jar:5.10.0:compile
        +-org.junit.platform:junit-platform-engine:jar:1.10.0:compile
          +-org.junit.platform:junit-platform-commons:jar:1.10.0:compile


```

Reply:
In the other modules I see it's done by excluding/including dependencies, but I'd rather use the dependencyManagement part of the pom.xml for this.
I'm pushing a proposal.